boards/nucleo32-f031/include/periph_conf.h

```diff
@@ -174,21 +174,6 @@ static const spi_conf_t spi_config[] = {
 #define RTC_NUMOF           (0U)
 /** @} */
 
-/**
- * @name   ADC configuration
- * @{
- */
-#define ADC_CONFIG {            \
```

Comment:
might be related to the fact that SPI1 and some PWM channels are on some of the ADC lines, too. But even the original ADC config does not ignore those pins and would hence collide with those, too.